rpmvenv/extensions/loader.py

```diff
@@ -21,10 +21,11 @@ class InvalidDependency(Exception):
 
 def load_extensions(whitelist=()):
     """Get an iterable of extensions in order."""
-    whitelist = tuple(set(('core',) + tuple(whitelist)))
+    whitelist = tuple(('core',) + tuple(whitelist))
+    unique_whitelist = tuple(set(whitelist))
```

Comment:
Yeah, you are right about that. I missed the fact we still use the whitelist var further down for sorting. Good call.